app/assets/javascripts/application.js

```diff
@@ -6,7 +6,8 @@
 //= require length_counter
 //= require markdown_preview
 //= require toggle_display_with_checked_input
-
+//= require govuk_publishing_components/dependencies
+//= require govuk_publishing_components/all_components
```

Comment:
tbh not sure i get which comment relates to removal of govspeak component

Reply:
looks like we probably still want the govspeak component even if just using html as it's really a component for consistent styling for html having been transformed from govspeak, otherwise we'll be manually doing design system markup

Reply:
Wouldn't have thought you'd need complex HTML though I'd expect you to be just using design system typography classes: https://design-system.service.gov.uk/styles/typography/